static/src/javascripts/projects/commercial/modules/dfp/on-slot-render.js

```diff
@@ -76,7 +81,7 @@ export const onSlotRender = (event: SlotRenderEndedEvent): void => {
         // Set refresh field based on the outcome of the slot render.
         const sizeString = advert.size && advert.size.toString();
         const isNotFluid = sizeString !== '0,0';
-        const isOutstream = sizeString === adSizes.outstream.toString();
+        const isOutstream = outstreamSizes.includes(sizeString);
```

Comment:
Same question as earlier. It feels a bit brittle to rely just on size. Maybe file a card to address that in the future if possible?

Reply:
Yes, I'll add a card for that.  It's a pity we can't have named sizes for special cases, apart from fluid.

Reply:
https://trello.com/c/M5zKo5I6